docs/RouterConfiguration.md

```diff
@@ -3,22 +3,23 @@
-| `routingDefaults`      | Default routing parameters, which will be applied to every request | object                 |               | see [routing defaults](#routing-defaults)                             |
-| `streetRoutingTimeout` | maximum time limit for street route queries                        | double                 | null          | units: seconds; see [timeout](#timeout)                               |
-| `requestLogFile`       | Path to a plain-text file where requests will be logged            | string                 | null          | see [logging incoming requests](#logging-incoming-requests)           |
-| `transit`              | Transit tuning parameters                                          | `TransitRoutingConfig` |               | see [Tuning transit routing](#Tuning-transit-routing)                 |
-| `updaters`             | configure real-time updaters, such as GTFS-realtime feeds          | object                 | null          | see [configuring real-time updaters](#configuring-real-time-updaters) |
-| `transmodelApi`        | configure Entur Transmodel API (**Sandbox**)                       | object                 | null          | See the code for parameters, no doc provided.                         |
+config key | description | value type | value default | notes
+---------- | ----------- | ---------- | ------------- | -----
+`routingDefaults` | Default routing parameters, which will be applied to every request | object |  | see [routing defaults](#routing-defaults)
+`streetRoutingTimeout` | maximum time limit for street route queries | double | null | units: seconds; see [timeout](#timeout)
+`requestLogFile` | Path to a plain-text file where requests will be logged | string | null | see [logging incoming requests](#logging-incoming-requests)
+`transit` | Transit tuning parameters | `TransitRoutingConfig` |  | see [Tuning transit routing](#Tuning-transit-routing)
+`updaters` | configure real-time updaters, such as GTFS-realtime feeds | object | null | see [configuring real-time updaters](#configuring-real-time-updaters)
+`transmodelApi` | configure Entur Transmodel API (**Sandbox**) | object | null | See the code for parameters, no doc provided.
+
 
 ## Routing defaults
 
 There are many trip planning options used in the OTP web API, and more exist
 internally that are not exposed via the API. You may want to change the default value for some of these parameters,
 i.e. the value which will be applied unless it is overridden in a web API request.
 
-A full list of them can be found in the [RoutingRequest](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java).
+A full list of them can be found in the [RoutingRequest](https://github.com/opentripplanner/OpenTripPlanner/blob/v2.0.0/src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java).
```

Comment:
Why do you change the URL to use the 2.0.0 tag?

Reply:
Reverted.